robottelo/ui/container.py

```diff
@@ -71,7 +71,22 @@ def create(self, resource_name, name, command, parameter_list):
                 {'main_tab_name': 'Environment', 'name': 'TTY', 'value': True},
             ]
         """
-        self.assign_value(locators['container.resource_name'], resource_name)
+        # send_keys() can't send left parenthesis (see SeleniumHQ/selenium#674)
```

Comment:
If this is a send_keys issue the fix should go before calling it, like replacing the ( with nothing or, which I think is best, fix the data we are passing in order to avoid using the `(` character. Look how many lines needed to workaround this here and if changing the data to remove `(` would be easier and faster.